Example/Example.Android/MainActivity.cs

```diff
@@ -23,9 +27,13 @@ protected override void OnCreate(Bundle savedInstanceState)
             base.OnCreate(savedInstanceState);
 
             global::Xamarin.Forms.Forms.Init(this, savedInstanceState);
+            
             CachedImageRenderer.Init(true);
             ImageCircleRenderer.Init();
-            LoadApplication(new App());
+            UserDialogs.Init(() => (Activity)Forms.Context);
+            var app = new App();
+            MobileCoreInitializer.Init(app);
```

Comment:
Initialization code requires App to determine execution Assemly and access resources where JSON configuration file is stored. I didn't find a better way. Is there any?